packages/medusa/src/models/publishable-api-key-sales-channel.ts

```diff
@@ -1,12 +1,22 @@
-import { Entity, PrimaryColumn } from "typeorm"
+import { BeforeInsert, Column, Entity } from "typeorm"
```

Comment:
question: Should this follow similar convention as with the OrderSalesChannel?

Reply:
agree, I updated the entity